README.md

```diff
@@ -316,26 +316,6 @@ Note that right now this code assumes you're installing everything for DetSys pu
 It is an explicit goal for this repository to support configuring things for *your* purposes without necessarily having to fork the repo.
 Please open issues discussing or send PRs improving this.
 
-#### Tailscale Token
-
-The tailscale token should be configured as follows.
-
-First configure a tag to assign to ephemeral macs, by adding this to your ACL:
-
-```json
-	"tagOwners": {
-		"tag:ephemeral-mac-ci": ["you@example.com"],
-	}
-```
```

Comment:
We probably want to keep this as documentation.

Reply:
This token now comes from Vault, so there's nothing left to _actually do_ for the setup here -- and this was quite clearly "task" documentation and not "how it works" documentation.

Reply:
I think that's not exactly true, and we do still want this: https://tailscale.com/kb/1068/acl-tags/#defining-a-tag says that you need to define the tag before you can use it (and somewhat relatedly, tagged machines don't expire since March 2022: https://tailscale.com/blog/tagged-key-expiry/).